### PR TITLE
Run nats locally

### PR DESCRIPTION
**Pull request by dgboss**
Rename nats.py modules within our codebase to avoid module resolution issues when running locally.
Add NATS.md
# Test Links:
[Percentile Calculator](https://wps-pr-2446.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-2446.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-2446.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-2446.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-2446.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-2446.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-2446.apps.silver.devops.gov.bc.ca/hfi-calculator)
[FWI Calculator](https://wps-pr-2446.apps.silver.devops.gov.bc.ca/fwi-calculator)

### CODE REVIEW

File: docs/NATS.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+*a. If there is no default VM*
+
+```bash
+limactl start --name=default template://docker
+```
+
+*b. If there is a default VM but it is not running*
+
+```bash
+lima start default
```
Comment (andrea-williams) [inline docs/NATS.md:28]:
```suggestion
limactl start default
```

**Review by andrea-williams — APPROVED** [review]
Unable to test this out fully (thanks Airdrop!), but Nats server appears to be running fine locally

**Review by conbrad — APPROVED** [review]
Awesome, thanks for this!

File: docs/NATS.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dgboss) [reply docs/NATS.md]: Thanks for catching that!

**Pull request merged**